test/ffmpeg-qsv/transcode/transcoder.py

```diff
@@ -10,42 +10,139 @@
 
 @slash.requires(have_ffmpeg)
 @slash.requires(have_ffmpeg_qsv_accel)
```

Comment:
please add `@slash.requires(using_compatible_driver)`

Reply:
added it